en/middleware/micrortps.md

```diff
@@ -34,6 +33,25 @@ The *Client* application is also compiled and built into the firmware as part of
 > **Tip** The bridge code can also be [manually generated](micrortps_manual_code_generation.md). Most users will not need to do so, but the linked topic provides a more detailed overview of the build process and can be useful for troubleshooting.
 
 
+## Supported uORB messages
```

Comment:
If this is going to happen in the coming weeks OR if this is 90% complete and we're only deciding on some outliers then I'd leave it as I had it - and just add a link to the existing location. The reason being that it is good to keep the docs as close to how it "should" eventually look. 

If everyone is going to have to edit this list, then you're quite right to move it. Which is true?